apps/api/v2/src/ee/bookings/2024-08-13/services/output.service.ts

```diff
@@ -444,4 +445,31 @@ export class OutputBookingsService_2024_08_13 {
       },
     };
   }
+
+  async getOutputBookingRecordings(recordings: GetRecordingsOfCalVideoByRoomNameOutput[]) {
```

Comment:
I was at first thinking of having cal video output service, but then saw that this function invokes `getDownloadLinkOfCalVideoByRecordingId` which seems like have a lot going on there.

Output service should simply transform data from format X to Y, which is why then i wanted to ask for the `getOutputBookingRecordings` to receive return of `getDownloadLinkOfCalVideoByRecordingId`, but given that we invoke core `getDownloadLinkOfCalVideoByRecordingId` and then output transformation is pretty short we can simply move code of `getOutputBookingRecordings` into [cal-video.service.ts](https://github.com/calcom/cal.com/pull/21775/files#diff-88a7c6e819c866e495cdb57128da1babde625fd6e8d7a417ea2859c153b67d7d) `getRecordings` function.

Reply:
And then we can remove `GetRecordingsOfCalVideoByRoomNameOutput` - output should only store what the api actually responds with to the client.